### DISCUSSION TITLE
Welcome to tock Discussions!

### DISCUSSION THREAD

**Discussion by alevy** [tock/tock#2247] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alistair23** [comment]
Ah this is cool!

**Comment by jrvanwhy** [comment]
I'm Johnathan Van Why. I am a Google employee on the [OpenTitan](https://opentitan.org/) project. To date, I have primarily been contributing to `libtock-rs`, trying to improve its reliability (via better testing infrastructure) and code size.

**Comment by lschuermann** [comment]
Hi, I'm Leon. I am a student at the University of Stuttgart, Germany. I'm interested in Rust, operating systems development and trying to bring networking support (IEEE 802 & IP) to Tock. I like contributing to free software.
